Gluntch, the Bestower.txt

```diff
@@ -0,0 +1,15 @@
+Name:Gluntch, the Bestower
+ManaCost:1 G W
+Types:Creature Jellyfish
+PT:0/5
+K:Flying
+SVar:X:PlayerCountPlayers$Amount

```

Comment:
I think move this down between the token and cleanup lines... most scripts put X lines like this somewhere below where they are referenced

anywhere down there is fine